references/cli/logs.go

```diff
@@ -77,6 +98,7 @@ func NewLogsCommand(c common.Args, order string, ioStreams util.IOStreams) *cobr
 	cmd.Flags().StringVarP(&largs.ClusterName, "cluster", "", "", "filter the pod by the cluster name")
 	cmd.Flags().StringVarP(&largs.PodName, "pod", "p", "", "specify the pod name")
 	cmd.Flags().StringVarP(&largs.ContainerName, "container", "", "", "specify the container name")
+	cmd.Flags().StringVarP(&largs.StepName, "step", "s", "", "specify the step name")
```

Comment:
Should we add tells that this argument has conflict with `--pod` or any other args?

Reply:
yes, we need and more explanation